packages/flutter/test/rendering/baseline_utils.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+class _PaintInstructionMatcher extends Matcher {
```

Comment:
Does this matcher provide a clear error message when it fails? I see a lot of our custom matcher implement describeMismatch to help out with that, but this one doesn't.